src/content/add-to-app/ios/project-setup.md

```diff
@@ -152,108 +156,103 @@ some/path/
-`MyApp` directory to create one. 
-You can find more details on using 
-CocoaPods in the [CocoaPods getting started guide][].
+#### Update your Podfile
 
-<ol>
-<li>
+1. If your existing application (`MyApp`) doesn't already have a Podfile,
+   use the `pod init` command in the `MyApp` directory to create one.
 
-Add the following lines to your `Podfile`:
+   To learn more about CocoaPods,
+   consult the [CocoaPods getting started guide][].
 
-```ruby title="MyApp/Podfile"
-flutter_application_path = '../my_flutter'
-load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
-```
+1. Add the following lines to your `Podfile`.
 
-</li>
+   ```ruby title="MyApp/Podfile"
+   flutter_application_path = '../my_flutter'
+   load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
+   ```
 
-<li>
+1. For each [Podfile target][] that needs to embed Flutter,
+   add a call to `install_all_flutter_pods(flutter_application_path)`.
 
-For each [Podfile target][] that needs to
-embed Flutter, call `install_all_flutter_pods(flutter_application_path)`.
+   ```ruby title="MyApp/Podfile"
+   target 'MyApp' do
+     install_all_flutter_pods(flutter_application_path)
+   end
+   ```
 
-```ruby title="MyApp/Podfile"
-target 'MyApp' do
-  install_all_flutter_pods(flutter_application_path)
-end
-```
+1. In the `Podfile`'s `post_install` block,
+   add a call to `flutter_post_install(installer)`.
 
-</li>
+   ```ruby title="MyApp/Podfile"
+   post_install do |installer|
+     flutter_post_install(installer) if defined?(flutter_post_install)
+   end
+   ```
 
-<li>
+   The `flutter_post_install` method adds build settings to support
+   native Apple Silicon `arm64` iOS simulators.
+   Include the `if defined?(flutter_post_install)`
+   check to validate your `Podfile` if you are running on older versions
+   of Flutter that lack this method.
 
-In the `Podfile`'s `post_install` block, call `flutter_post_install(installer)`.
+1. Use the `flutter pub get` command to refresh plugins.
 
-```ruby title="MyApp/Podfile"
-post_install do |installer|
-  flutter_post_install(installer) if defined?(flutter_post_install)
-end
-```
+   When you change the Flutter dependencies in the `pubspec.yaml` file,
+   run `flutter pub get` in your Flutter module directory.
+   This refreshes the list of plugins that the `podhelper.rb` script reads.
 
-:::note
-The `flutter_post_install` method (added in Flutter 3.1.0),
-adds build settings to support native Apple Silicon `arm64` iOS simulators.
-Include the `if defined?(flutter_post_install)` check to ensure your `Podfile`
-is valid if you are running on older versions of Flutter that don't have this method.
-:::
+1. Run the `pod install` command to embed the plugins and frameworks.
 
-</li>
+   From your iOS app project at `/path/to/MyApp`, run `pod install`.
 
-<li>
+   The `podhelper.rb` script embeds your plugins,
+   `Flutter.framework`, and `App.framework` into your project.
```

Comment:
Should we move this up to the step that adds the `podhelper.rb` script to your `Podfile`?